search/django/weedid/utils.py

```diff
@@ -118,3 +128,40 @@ def retrieve_listing_info(query_entity, awaiting_review):
         "contributor": query_entity.user.username,
         "contributor_email": query_entity.user.email if awaiting_review else "",
     }
+
+
+def send_email(subject, body, recipients):
+    with smtplib.SMTP(SMTP_HOST, SMTP_PORT) as smtp:
+        for recipient in recipients:
+            msg = EmailMessage()
+            msg["Subject"], msg["From"], msg["To"] = subject, SERVER_ADDR, recipient
+            msg.set_content(body)
+            smtp.send_message(msg)
+
+
+def upload_notification(upload_id):
```

Comment:
Please pull this into a different module, not `utils.py`, e.g. `notification.py`